autoware_control_msgs/msg/Lateral.msg

```diff
@@ -0,0 +1,22 @@
+# Lateral control message for Ackermann-style platforms
+#
+# Note regarding tires: If the platform has multiple steering tires, the commands
+# given here are for a virtual tire at the average lateral position of the steering tires.
+#
+# Look up Ackermann Bicycle Model for more details
+#
+# Positive values represents left inclination (if forward)
+# Negative values represents right inclination (if forward)
+#
+# Illustration: https://autowarefoundation.github.io/autoware-documentation/pr-79/design/autoware-interfaces/components/vehicle-dimensions/#vehicle-dimensions_1
```

Comment:
update this link

Reply:
moved it out of the message in https://github.com/autowarefoundation/autoware_msgs/pull/11/commits/57d594b697c47260c3c755e77737f39edeed8d29